pkg/rid/application/subscription_test.go

```diff
@@ -77,10 +75,9 @@ func (store *subscriptionStore) DeleteSubscription(ctx context.Context, s *ridmo
 		delete(store.subs, s.ID)
 		return sub, nil
 	}
-	return nil, sql.ErrNoRows
```

Comment:
Let's clarify the expected behavior of the interface indicating that nil should be returned when the thing wasn't found (rather than returning sql.ErrNoRows), and repeat for ISAs as well.  Are there any other behavioral requirements of the interface that aren't documented on the interface currently?

Reply:
done